CefSharp.Core/Internals/CefRegisterCdmCallbackAdapter.h

```diff
@@ -0,0 +1,53 @@
+// Copyright � 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "Stdafx.h"
+#include "CefWrapper.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        private class CefRegisterCdmCallbackAdapter : public CefRegisterCdmCallback
+        {
+        private:
+            gcroot<IRegisterCdmCallback^> _callback;
+
+        public:
+            CefRegisterCdmCallbackAdapter(IRegisterCdmCallback^ callback)
+            {
+                _callback = callback;
+            }
+
+            ~CefRegisterCdmCallbackAdapter()
+            {
+                if (static_cast<IRegisterCdmCallback^>(_callback) != nullptr)
```

Comment:
You can call `delete` on a `nullptr`, ends up being a `noop`, so no need to check this.